packages/trace-viewer/src/ui/actionList.tsx

```diff
@@ -104,6 +104,29 @@ export const ActionList: React.FC<ActionListProps> = ({
   </div>;
 };
 
+const ActionParameterContext: React.FC<{
```

Comment:
Style nit: as Context does not contribute to understanding of what that means, drop the suffix.